runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.cc

```diff
@@ -159,6 +159,10 @@ AMDAIEDeviceModel::AMDAIEDeviceModel(
   TRY_XAIE_API_FATAL_ERROR(XAie_TurnEccOff, &devInst);
 }
 
+uint8_t AMDAIEDeviceModel::getAddressGranularityInBits() const {
```

Comment:
bruh lol - this is the [goofiest name](https://github.com/Xilinx/mlir-aie/pull/1538#discussion_r1629938744) for `MinStrideBitWidth`.

Reply:
Yeah, I just named this in consistency with aie-rt/mlir-aie I believe, but yeah, it's goofy. Your suggestion is way better.. Updated it.